demo/device/README.md

```diff
@@ -0,0 +1,13 @@
+# Getting the executable
+
+Use following commands to build FIDO IoT HTTP Device Component sample source.
+```
+$ cd <fido-iot-src>/service/component-samples/http-device-sample/
+$ mvn clean install
+```
+
+This will copy the required executables and libraries into <fido-iot-src>/demo/device/.
+
+# Starting the Device service
+
+Refer the [Device Component Samples](../../README.md/#running-the-fido-iot-http-java-device-sample) section to start the device.
```

Comment:
The demo component would also be released as a stand-alone binary. We should be including all necessary information without having to refer to other source README files.

A link to demo README from other source README should be okay, though.

Reply:
Done